app/main.py

```diff
@@ -1,6 +1,24 @@
+import os
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+COUNTRY = "France"
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set in your environment. Without it, the API request will fail.